src-opam/distro.ml

```diff
@@ -868,6 +879,7 @@ let tag_of_distro (d : t) =
   | `Cygwin v -> "cygwin-" ^ win10_release_to_string v
   | `Windows (`Mingw, v) -> "windows-mingw-" ^ win10_release_to_string v
   | `Windows (`Msvc, v) -> "windows-msvc-" ^ win10_release_to_string v
+  | `Macos (`Latest | `Macos_monterey) -> "macos-homebrew-12.6"
```

Comment:
I don't think we should store patches versions for macos as there are usually no significant difference and it changes too often.
```suggestion
  | `Macos (`Latest | `Macos_monterey) -> "macos-homebrew-12"
```